.github/workflows/publish_container.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Container Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_version:
+        description: "Image version"
+        default: "dev"
+        type: string
+        required: False
+  workflow_call:
+    inputs:
+      image_version:
+        description: "Image version"
+        default: "dev"
```

Comment:
Why the default should be `dev`?
Shouldn't it be a reference to the tag used when calling this `yml` file or in other words: `${{ github.ref_name }}`

Reply:
The default is dev, once the most common case will be publishing the main (our `dev` tag) for merged commits.

Reply:
But isn't `${{ github.ref_name }}` the same as `dev`? 
I read it's the commit sha of the commit starting the workflow.
If there are equal probably `dev` is clearer, but just wanted to make sure I got this correct.